my-next-app/src/app/scim/v2/Users/[[...params]]/route.ts

```
@@ -0,0 +1,191 @@
+      return NextResponse.json(user, { status: 200 });
+    } else {
+      console.log(`SCIM Users GET: User ${userId} not found`);
+      return NextResponse.json({ schemas: ["urn:ietf:params:scim:api:messages:2.0:Error"], detail: 'User not found' }, { status: 404 });
+    }
+  } else {
+    // List users
+    // Basic filtering support for userName eq (as per many IdP checks)
+    const filter = queryParams.get('filter');
+    let resources = Object.values(usersDB);
+
+    if (filter && filter.includes('userName eq')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)